pipelines/ApPipe.yaml

```diff
@@ -65,8 +65,8 @@ contracts:
       transformDiaSrcCat.connections.ConnectionsClass(config=transformDiaSrcCat).diaSourceCat.name
   - imageDifference.connections.ConnectionsClass(config=imageDifference).subtractedExposure.name ==
         diaPipe.connections.ConnectionsClass(config=diaPipe).diffIm.name
-  - imageDifference.connections.ConnectionsClass(config=imageDifference).warpedExposure.name ==
-        diaPipe.connections.ConnectionsClass(config=diaPipe).warpedExposure.name
+  - imageDifference.connections.ConnectionsClass(config=imageDifference).template.name ==
+        diaPipe.connections.ConnectionsClass(config=diaPipe).template.name
```

Comment:
Can the `doWriteWarpedExp` override be removed from `imageDifference` (and likewise in `ApPipeWithFakes`)?